.github/workflows/markdown_link_checker.yml

```diff
@@ -1,5 +1,10 @@
 on: [pull_request]
```

Comment:
you need to delete this line as well. There are two `on:` now

Reply:
@RishiKumarRay This is still not a valid workflow. This is what I wanted:


```
name: Check links for markdown files
on:
    push:
        branches:
            - main
    pull_request:
jobs:
  markdown-link-check:
    runs-on: ubuntu-latest
    steps:
    - uses: actions/checkout@v2
    - uses: gaurav-nelson/github-action-markdown-link-check@v1
      with:
        use-quiet-mode: 'yes'
```

Reply:
@srknzl check now